content/en/logs/guide/logs-rbac-permissions.md

```diff
@@ -110,25 +110,22 @@ Assign the role(s) in the modal of a specific pipeline.
 
 1. [Get the Roles ID][1] of the role you want to assign to specific pipelines.
 2. [Get the Permission ID][2] for the `logs_write_processors` permission API for your region.
-3. [Get the Pipeline ID(s)][3] of the pipeline(s) you want to assign this role on.
-4. Grant permission to that role with the following call:
+3. Grant permission to that role with the following call:
 
 ```sh
 curl -X POST \
-        https://app.datadoghq.com/api/v1/role/<ROLE_UUID>/permission/<PERMISSION_UUID> \
+        https://app.datadoghq.com/api/v2/roles/<ROLE_UUID>/permissions \
         -H "Content-Type: application/json" \
         -H "DD-API-KEY: <YOUR_DATADOG_API_KEY>" \
         -H "DD-APPLICATION-KEY: <YOUR_DATADOG_APPLICATION_KEY>" \
         -d '{
-                "scope": {
-                    "pipelines": [ "<PIPELINE-X_ID>", "<PIPELINE-Y_ID>"]
-                }
+                "id": "<PERMISSION_UUID>",
+                "type": "permissions"
             }'
 ```
 
 [1]: /api/v2/roles/#list-roles
```

Comment:
These two links appear to be broken. If you have time, can you update them?

Reply:
I'll approve and merge this PR so the changes can go live, but if you have a chance, please submit a follow-up PR to fix the links.

Reply:
Never mind, the links work on the production site. I'm not sure why they don't work in the preview.